sherpa/astro/ui/tests/test_astro_ui_utils_simulation.py

```diff
@@ -50,6 +51,7 @@ def test_fake_pha_no_rmf(idval, clean_astro_ui):
         ui.fake_pha(idval, arf=None, rmf=None, exposure=1000.0)
 
 
+@requires_fits
```

Comment:
How did this pass without "requires_fits" before? It did, so why do we need to make this as requires_fits now?

Reply:
I'd have to check. There's a possibility that the code paths have changed so that it now ends up calling a routine from `sherpa.astro.io` when it did not before, but it's been so long since I looked at this in detail that I just don't remember,

Alternatively the addition isn't needed....

Reply:
Aha. It is exactly because I've removed a check for file existence in `unpack_arf`/`unpack_rmf` (which would always be run) because it's also a part of the I/O code (but then it can only run in there is actually a working backend).

So there is the question of whether this simplification is worth it (essentially - do we want the second commit)?

Reply:
OK, understood. Let's keep that change. It just was not obvious to me, so I figured I'll ask you.